test/pebble/src/test/java/org/glassfish/ozark/test/pebble/PebbleEngineProducerTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright © 2017 Ivar Grimstad (ivar.grimstad@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.glassfish.ozark.test.pebble;
+
+import com.google.common.cache.Cache;
+import com.mitchellbosecke.pebble.PebbleEngine;
+import com.mitchellbosecke.pebble.cache.BaseTagCacheKey;
+import com.mitchellbosecke.pebble.extension.NodeVisitor;
+import com.mitchellbosecke.pebble.extension.escaper.EscapeFilter;
+import com.mitchellbosecke.pebble.loader.ServletLoader;
+import com.mitchellbosecke.pebble.template.PebbleTemplate;
+import com.mitchellbosecke.pebble.template.PebbleTemplateImpl;
+import java.util.Locale;
+import java.util.Properties;
+import java.util.stream.IntStream;
+import org.glassfish.ozark.ext.pebble.PebbleEngineProducer;
+import org.glassfish.ozark.ext.pebble.PebbleProperty;
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PebbleEngineProducerTest {
+
+  Properties properties;
+  PebbleEngineProducer pebbleEngineProducer;
```

Comment:
\[Checkstyle\] ERROR: Variable 'pebbleEngineProducer' must be private and have accessor methods\.